.github/pull_request_template.md

```diff
@@ -1,39 +1,52 @@
-## Description
+### Description
 
-A detailed explanation of the changes in your PR.
+<!-- A detailed explanation of the changes in your PR. Feel free to remove this section if the title of your PR is sufficiently descriptive. -->
 
-Feel free to remove this section if it is overkill for your PR, and the title of your PR is sufficiently descriptive.
+change me!
 
-## Checklist
-- [ ] Investigated and inspected CI test results
-- [ ] Unit test and regression tests added
-- [ ] Evaluated and added CHANGELOG entry if required
-- [ ] Determined and documented upgrade steps
-- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))
+### User-facing documentation
 
-If any of these don't apply, please comment below.
+(*must be* 2 items and both *must be* checked)
+<!-- Remove conflicting items that won't be checked. -->
 
-## Testing Performed
+- [ ] CHANGELOG is updated
+- [ ] CHANGELOG update is not needed
+- [ ] Documentation PR is created and linked above
+- [ ] Documentation is not needed
 
-### Here I tell how I validated my change
+### Testing
 
-TODO(replace-me)  
+- [ ] inspected CI results
+
+#### Automated testing
+
+(*must be* at least 1 item and all items *must be* checked)
+<!-- Remove item(s) that don't apply and won't be checked. -->
+
+- [ ] added unit tests
+- [ ] added e2e tests
+- [ ] added regression tests
+- [ ] added compatibility tests
+- [ ] modified existing tests
+- [ ] contributed **no automated tests**
```

Comment:
Why there is no `- [ ] existing tests are sufficient`?

Reply:
Maybe because the author and reviewer(s) did not see how it's different from `- [ ] contributed **no automated tests**`.